koiic/vehicle-store-api
# #4 An endpoint to update a car instance

**What does this PR do?**

An endpoint to retrieve a single car object

**Description of Task to be completed?**

As a user, I should be able to update a single car object

**How should this be manually tested?**

- Fetch the branch with `git fetch origin ft-update-single-car-04 `
- Check out the branch with `git checkout ft-update-single-car-04 `
- Run the app: `docker-compose up` or `python manage.py runserver`
- On postman create a get request to this url `http://127.0.0.1:8000/api/car/<int:id>/`

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

N/A

**Screenshots (if appropriate)**

N/A
